src/libraries/System.Text.RegularExpressions/tests/RegexSRMTests.cs

```diff
@@ -29,6 +29,148 @@ static void WriteLine(string s)
         private const char Turkish_i_withoutDot = '\u0131';
         private const char Kelvin_sign = '\u212A';
 
+        /// <summary>
+        /// Maps each character c to the set of all of its equivalent characters if case is ingored or null if c in case-insensitive
```

Comment:
```suggestion
        /// Maps each character c to the set of all of its equivalent characters if case is ignored or null if c in case-insensitive
```